index.html

```diff
@@ -1,71 +1,268 @@
+            <li>
+              <a href="#"
+                ><img src="images/github.svg" alt="Github icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="images/twitter.svg" alt="Peace icon"
+              /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <!--End of Section-->
+    </div>
 
-        <!--Start:Section-->
-        <section class="main">
-            <div class="main-section">
-                <h1 class="intro-name">I’m Anshuman.<br>Glad to see<br> you!</h1>
-                <p class="intro-bio">I’m a software developer! I can help you <br>
-                     build a product , feature or website Look<br> through some of my work and<br>
-                      experience! If you like what you see and<br>
-                       have a project you need coded, don’t<br>
-                        hesitate to contact me.</p>
-                <h3 class="social-text">LET’S CONNECT</h3>
-                <div class="social-media">
-                        <ul class="horizontal-list">
-                            <li>
-                                <a href="#">
-                                    <img src="images/twitter.svg" alt="twitter">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="">
-                                    <img src="images/Linkedin.svg" alt="Linkedin">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="">
-                                    <img src="images/microverse.svg" alt="Meta">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="">
-                                    <img src="images/github.svg" alt="github">
-                                </a>
-                            </li>
-                            <li>
-                                <a href="">
-                                    <img src="images/victory.svg" alt="victory">
-                                </a>
-                            </li>
-                        </ul>
-                </div>
-                
-            </div>
-        </section>
-       <!--End of Section-->
+    <!--Starting  of work  section-->
+    <section class="work-section">
+      <!--First Card starting-->
+      <div class="card">
+        <img src="images/Snapshoot Portfolio.svg" alt="Project Screenshot" />
+        <div class="card-text">
+          <h2>Tonic</h2>
+          <p>
+            <strong>CANOPY</strong>
+            <i class="fa-solid fa-circle fa-2xs"></i> Back End Dev
+            <i class="fa-solid fa-circle fa-2xs"></i> 2015
+          </p>
+          <p class="detail">
+            A daily selection of privately personalized reads; no accounts or
```

Comment:
- [ ] Wonderful job so far. please kindly fix the fixed width and make your project responsive up to the desktop version(768px)

![Screenshot (1105)_LI](https://github.com/anshuman7negi/my-portfolio/assets/65298048/e43153ae-cc87-4628-bfe3-b70533001b52)